dev/tools/controllerbuilder/cmd/codebot/main.go

```diff
@@ -70,6 +73,9 @@ func (cb *CodeBot) run(ctx context.Context) error {
 	flag.StringVar(&o.BaseDir, "base-dir", o.BaseDir, "base directory for the project code")
 	flag.StringVar(&o.Prompt, "prompt", o.Prompt, "prompt to be passed in non-interactive mode")
 	flag.StringVar(&o.UIType, "ui-type", o.UIType, "available value is terminal, tview, prompt or bash.")
+	flag.StringVar(&o.ToolFilterInclude, "tool-filter-include", o.ToolFilterInclude, "Include specific tools. If not set, default to include all tools. e.g. `File` to include all file related oprations but not tools. `Workspace` to include Workspace operations. `File|Workspace` to include both.")
```

Comment:
Nit: Typos in `e.g. "File" to include all file related oprations but not tools`? I guess it's `operations` and `but not other tools`?